suite/runsuite.cmake

```diff
@@ -555,9 +555,10 @@ endif ()
 if (ARCH_IS_X86 AND UNIX)
   # TODO i#3544: Run tests under QEMU
   set(orig_extra_ctest_args ${extra_ctest_args})
+  set(extra_ctest_args INCLUDE_LABEL RISCV64)
```

Comment:
Add a comment that this is temporary with `TODO i#3544: Get all the tests working`.